activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Specify callback in `has_secure_token`
+
+    ```ruby
+    class User < ApplicationRecord
+        has_secure_token on: :initialize
```

Comment:
remove extra indentation (2 spaces) in this line

Reply:
@guilleiguaran done!